recipe/meta.yaml

```
@@ -1,20 +1,22 @@
 {% set name = "gstlal" %}
-{% set version = "1.5.0" %}
-{% set sha256 = "87077e24d62b63f500f51194124fb203ed21381242ee241aa13887b1f9de6d07" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "92926bfc692adea418b8aaaf87663eae10970c78bf4dd0e2fd36c1b68c43392f" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
```

Quoting here breaks the update bot (or at least used to). It's generally safer to remove the variable:
```suggestion
  sha256: 92926bfc692adea418b8aaaf87663eae10970c78bf4dd0e2fd36c1b68c43392f
```